airbrake/handler.py

```diff
@@ -25,7 +25,7 @@ class AirbrakeHandler(logging.Handler):
     # pylint: disable=too-many-arguments
 
     def __init__(self, airbrake=None, level=logging.ERROR, project_id=None,
-                 api_key=None, environment=None, base_url=None, hostname=None):
+                 api_key=None, **config):
```

Comment:
Ah, I see. FWIW I don't think a grab-bag variable is a good idea. I would recommend against using a single variable for this *and* for configuring the client. There are other designs/approaches to consider, and I can throw out those ideas if you would like